desktop/core/src/desktop/conf.py

```diff
@@ -2588,7 +2594,7 @@ def get_ldap_bind_password(ldap_config):
 GS_BULK_DELETE_DIR_KEYS_MAX_LIMIT = Config(
   help=_('Maximum number of keys with specific directory prefix that can be deleted in a single bulk operation in GS.'),
   key='gs_bulk_delete_dir_keys_max_limit',
-  default=100,
+  default=1000,
```

Comment:
What is the constraint on this value? And in GS rmtree function, it deletes in a loop and and counting the number. Why the deletion interrupted in the middle but not at the beginning. User will confused and has clue to keep track of what are deleted and what else are not deleted?
https://github.com/cloudera/hue/blob/fced65e62b1ea83f5dc46e3ac7b728833f22c9d0/desktop/core/src/desktop/lib/fs/gc/gs.py#L317

Reply:
Removing this max limit. I tested out 5k key deletion in my local setup and there was no timeout and user could do other operations in the same browser.

Checked with the RAZ + HDFS CLI implementation and they are also not having any limit.